css/reset.css

```diff
@@ -0,0 +1,35 @@
+*{
+  margin: 0;
+  padding: 0;
+}
+ul,
+ol{
+  list-style: none;
+}
```

Comment:
Here and below 1. add empty line between each rule. 2. spaces between selectors and '{'.
Example:
```
* {
  margin: 0;
  padding: 0;
}

ul,
ol {
  list-style: none;
}
```